package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "scripts": {
+    "check-markdown": "node tools/markdown-checker/index.js",
+    "test": "jest --no-cache --verbose",
+    "test:coverage": "jest --no-cache --verbose --coverage"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "jest --no-cache --verbose"
```

Comment:
please remove this; husky is inherently dangerous. git hooks shouldn't be automatically installed, it's a security issue.